aether-entity-extraction-module/conf/pip/primary-requirements.txt

```diff
@@ -18,7 +18,6 @@ aether.python
 coverage
 fakeredis
 flake8-quotes
-flask
```

Comment:
do not forget to follow the instructions :grimacing: 

`docker-compose run --rm --no-deps exm pip_freeze`